src/Frontend/src/components/messages/SagaDiagram.spec.ts

```diff
@@ -350,8 +349,16 @@ function rendercomponent({ initialState = {} }: { initialState?: { MessageStore?
 
         const sagaUpdatesTimestampsValues = sagaUpdatesTimestamps.map((item) => item.innerHTML);
 
-        // //check if the values are in the same order as the sagaUpdates array passed to this function
-        expect(sagaUpdatesTimestampsValues).toEqual(sagaUpdates.map((item) => item.expectedRenderedLocalTime));
+        // Parse the rendered timestamp strings back to Date objects for comparison
+        const parsedDatesFromUI = sagaUpdatesTimestampsValues.map((timestampString) => {
+          // Parse the retrieved timestamp string  back to a Date
+          return new Date(timestampString);
```

Comment:
How will `new Date(string)` reliably parse localized strings like "04/03/2025 03:04 PM" or "03/04/2025 03:04 PM"?
Either could be 4 Mar or 3 Apr depending on parsing.

Reply:
We can use a set of expectations that are not susceptible to that. Could you update the constants?